### PR TITLE
Updated demo and code examples to use reactive forms and to disable a button dynamically

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
#30

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-reactive-form-demos](https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-reactive-form-demos)

File: src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html [hunk 1]
```diff
@@ -30,21 +30,27 @@
...
-    />
-  </div>
+      <label
+      class="sky-control-label sky-control-label-required"
+      [for]="firstNameInput.id"
+      >
+        First name
+      </label>
+      <input
+          class="sky-form-control"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html]: Code style, tabs.

File: src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html [hunk 2]
```diff
@@ -30,21 +30,27 @@
...
-    <input
-      class="sky-form-control"
-      skyId
-      type="text"
-      [(ngModel)]="demoModel.firstName"
-      #firstName="skyId"
-    />
-  </div>
+      <label
+      class="sky-control-label sky-control-label-required"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html]: Code style, tabs.

File: src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.ts [hunk 3]
```diff
@@ -14,34 +20,67 @@ import {
...
+      }
     ]
   };
 
   public showForm: boolean = false;
 
-  constructor() {
-    this.demoModel.firstName = this.firstName;
+  constructor(private formBuilder: FormBuilder) {
+    this.myForm = this.formBuilder.group({
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.ts:59]: I'd throw this code in a `ngOnInit` (at least) to demonstrate better practices.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
Might be worth implementing input-box for all of the form fields since Paul mentioned that would be a new requirement for code examples.

File: src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html]: Wow. Not sure what happened on both of these. Fixed.

File: src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html]: Fixed

File: src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.ts (hunk 3 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.ts:59]: Done on all of them

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-reactive-form-demos](https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-reactive-form-demos)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**